### PR TITLE
Timeout message for build analytics

### PR DESCRIPTION
**Pull request by brunobat**

### CODE REVIEW

File: independent-projects/tools/analytics-common/src/main/java/io/quarkus/analytics/AnalyticsService.java [hunk 1]
```diff
@@ -129,6 +131,14 @@ public void close() {
                 log.debug("[Quarkus build analytics] Build analytics sent successfully. Sent event can be seen at .../target/" +
                         fileLocations.lastTrackFileName());
             }
+        } catch (ExecutionException | TimeoutException e) {
+            if (log.isDebugEnabled()) {
+                log.debug("[Quarkus build analytics] Failed to send build analytics to Segment: " +
+                        e.getClass().getName() + ": " +
+                        (e.getMessage() == null ? "(no message)" : e.getMessage()));
+            }
+            log.info("[Quarkus build analytics] Failed to send build analytics to Segment. " +
```
Comment (cescoffier) [inline independent-projects/tools/analytics-common/src/main/java/io/quarkus/analytics/AnalyticsService.java:140]: Why info?

Comment (brunobat) [reply independent-projects/tools/analytics-common/src/main/java/io/quarkus/analytics/AnalyticsService.java:140]: A couple of weeks ago ppl mentioned that WARN would scare ppl without a reason.

Comment (brunobat) [reply independent-projects/tools/analytics-common/src/main/java/io/quarkus/analytics/AnalyticsService.java:140]: I guess it was useful while we were testing it but now it is just be an annoyance. 
Will remove that and the remote config download message.

Comment (maxandersen) [reply independent-projects/tools/analytics-common/src/main/java/io/quarkus/analytics/AnalyticsService.java:140]: fine to have it as debug. but yes; not shown by default as it confuses more than helping.

**Review by cescoffier — APPROVED** [review]

**Comment by brunobat** [comment]
@geoand can we merge this please? 
Some CI builds relying on message parsing might be broken by this message.

**Comment by geoand** [comment]
No need to ask if someone has already approved, no concerns have been risen and CI is green :)

Push the button 😉

**Pull request merged**